functions/src/webhooks/transcription.ts

```diff
@@ -0,0 +1,53 @@
+import * as functions from "firebase-functions"
+import { AssemblyAI } from "assemblyai"
+import { db } from "../firebase"
+import { Hearing } from "../events/types"
+import { sha256 } from "js-sha256"
+
+const assembly = new AssemblyAI({
+  apiKey: process.env.ASSEMBLY_API_KEY ? process.env.ASSEMBLY_API_KEY : ""
+})
+
+export const transcription = functions.https.onRequest(async (req, res) => {
```

Comment:
Not sure if this is an issue in practice given the time it takes to transcribe, but should we be returning some kind of error if we get a webhook message for a transcript we don't recognize (so AssemblyAI can retry)? 

i.e. A `404` if we can't find an event with a matching `videoAssemblyId`

Reply:
good call, added a 404.